packages/react-menu/src/components/MenuItem/useMenuItemStyles.ts

```diff
@@ -1,10 +1,11 @@
 import { mergeClasses, makeStyles } from '@fluentui/react-make-styles';
-import { createFocusIndicatorStyleRule } from '@fluentui/react-tabster';
+import { createFocusOutlineStyle } from '@fluentui/react-tabster';
 import type { MenuItemState } from './MenuItem.types';
 
 const useStyles = makeStyles({
-  focusIndicator: createFocusIndicatorStyleRule(),
+  focusIndicator: theme => createFocusOutlineStyle(theme),
   root: theme => ({
```

Comment:
There is no sense to have this as a separate rule, we can simply spread into `root`:

```diff
-  focusIndicator: theme => createFocusOutlineStyle(theme),
  root: theme => ({
+  ...createFocusOutlineStyle(theme)
```